frontend/src/app/testResults/submissions/Submissions.tsx

```diff
@@ -77,11 +78,11 @@ const Submissions = () => {
   };
 
   return (
-    <div className="grid-row">
+    <div className="grid-row header-size-fix">
       <div className="prime-container card-container">
         {/*Submission header*/}
         <div className="usa-card__header">
-          <h2>Upload history</h2>
+          <h1>Upload history</h1>
         </div>
 
         {/*filters*/}
```

Comment:
Another accessibility request here, I'm afraid - can you change these `DatePicker` elements to use the native `input type=date` element instead? The DatePicker component isn't accessible because of the modal thing it tries to show on screen (it's not a true modal, but also isn't read correctly by screenreaders)

Reply:
done.